pmlc/.clang-format

```diff
@@ -1,2 +1,2 @@
 BasedOnStyle: LLVM
-AlwaysBreakTemplateDeclarations: Yes
+AlwaysBreakTemplateDeclarations: true
```

Comment:
I copied this file directly from MLIR, why do we need to change it?

Reply:
This was giving me an error during the pre-commit check, "invalid boolean type" or something like that.

Reply:
You probably have the wrong version of clang format.  Are you running the precommit hooks from the conda env?

Reply:
Are you running `git commit` from the `.cenv` conda environment?

Reply:
Yes :/

Reply:
(In case anyone runs into this in the future, I believe my issue was that I'd configured & activated .cenv from the `master` branch then was trying to use that to commit to this branch off `plaidml-v1`. Resolved by removing .cenv then recreating it with `./configure` on `plaidml-v1`)